### ISSUE TITLE
Update Guzzle to v6

### ISSUE THREAD

**Issue by markdegrootnl** [fabulator/Fitbit#1]
Would it be possible to update guzzle to version 6?

**Comment by fabulator** [comment]
Yes, I plan to do some updated and this would be one of them, but I could not say how long it could take.
